itou/templates/signup/prescriber_orienter.html

```diff
@@ -0,0 +1,40 @@
+{% extends "layout/content_small.html" %}
+{% load bootstrap4 %}
```

Comment:
OMG `orienter` is a word! :-D

Il y a aussi `orientator`.

![image](https://user-images.githubusercontent.com/10533583/79000521-7135a880-7b55-11ea-8f43-770c424d69de.png)